test_integration_file_metadata_svc.sh

```diff
@@ -4,22 +4,35 @@
 GET_STATUS_SIGNAL=0
 CREATE_FILE_SIGNAL=1
 LIST_FILES_SIGNAL=2
+DEL_FILE_SIGNAL=3
 
 # Test variables for CreateFileMetadata
 VALID_A=0
 VALID_B=1
 NULL=2
 GARBAGE_UUID=3
 
-#Test variables for ListFileMetadataCollection
+# Test variables for ListFileMetadataCollection
 VALID_UUID_A=4
 VALID_UUID_B=5
 INVALID_UUID=6
+NON_EXISTENT_UUID=7
+
+# Test variables for DeleteFileMetadataCollection
+# INITIALIZE FIRST
+DEL_UUID="6ff30392-8ec8-45a4-ba94-5e22c4a686dc"
+DEL_FUID="5bcfa7cf667e56a778387a1c"
```

Comment:
for FUID, i feel like it makes more sense to say "unique file id" (UFID), but i see that if both started with a U, it can look the same. Then can we drop the U for unique? B/c ID can imply uniqueness (in html world lol). Small matter, I don't care either way, just a comment.